src/config/analyticsEvents.ts

```diff
@@ -0,0 +1,124 @@
+  VIEW_SWAP_TIMEOUT = "loaded screen: swap timeout",
+  VIEW_SWAP_SETTINGS = "loaded screen: swap settings",
+  VIEW_SWAP_CONFIRM = "loaded screen: swap confirm",
+  VIEW_SETTINGS = "loaded screen: settings",
+  VIEW_PREFERENCES = "loaded screen: preferences",
+  VIEW_CHANGE_NETWORK = "loaded screen: manage network",
+  VIEW_NETWORK_SETTINGS = "loaded screen: network settings",
+  VIEW_SHARE_FEEDBACK = "loaded screen: leave feedback",
+  VIEW_ABOUT = "loaded screen: about",
+  VIEW_SECURITY = "loaded screen: security",
+  VIEW_SHOW_RECOVERY_PHRASE = "loaded screen: show recovery phrase",
+  VIEW_MANAGE_CONNECTED_APPS = "loaded screen: manage connected apps",
+  VIEW_MANAGE_ASSETS = "loaded screen: manage assets",
+  VIEW_ADD_ASSET = "loaded screen: add asset",
+  VIEW_MANAGE_WALLETS = "loaded screen: manage wallets",
+  VIEW_IMPORT_SECRET_KEY = "loaded screen: import secret key",
+  VIEW_BUY_XLM = "loaded screen: add fund",
+  VIEW_SEARCH_ASSET = "loaded screen: search asset",
+  VIEW_ADD_ASSET_MANUALLY = "loaded screen: add asset manually",
+
+  // User Action Events
+  CREATE_PASSWORD_SUCCESS = "account creator: create password: success",
+  CREATE_PASSWORD_FAIL = "account creator: create password: error",
+  VIEWED_RECOVERY_PHRASE = "account creator: viewed phrase",
+  CONFIRM_RECOVERY_PHRASE_SUCCESS = "account creator: confirm phrase: confirmed phrase",
+  CONFIRM_RECOVERY_PHRASE_FAIL = "account creator: confirm phrase: error confirming",
+
+  // Authentication Events
+  RE_AUTH_SUCCESS = "re-auth: success", // Used for both confirm password and unlock
+  RE_AUTH_FAIL = "re-auth: error", // Used for both confirm password and unlock
+  RECOVER_ACCOUNT_SUCCESS = "recover account: success",
+  RECOVER_ACCOUNT_FAIL = "recover account: error",
+
+  // Send Payment Events
+  SEND_PAYMENT_SUCCESS = "send payment: payment success",
+  SEND_PAYMENT_PATH_PAYMENT_SUCCESS = "send payment: path payment success",
+  SEND_PAYMENT_FAIL = "send payment: error",
+  SEND_PAYMENT_SET_MAX = "send payment: set max",
+  SEND_PAYMENT_TYPE_PAYMENT = "send payment: selected type payment",
+  SEND_PAYMENT_TYPE_PATH_PAYMENT = "send payment: selected type path payment",
+  SEND_PAYMENT_RECENT_ADDRESS = "send payment: recent address",
+  SWAP_SUCCESS = "swap: success",
+  SWAP_FAIL = "swap: error",
+
+  // Copy Events
+  COPY_PUBLIC_KEY = "viewPublicKey: copied public key",
+  COPY_BACKUP_PHRASE = "backup phrase: copied phrase",
+  DOWNLOAD_BACKUP_PHRASE = "backup phrase: downloaded phrase",
+
+  // Transaction & Simulation Events
+  SIMULATE_TOKEN_PAYMENT_ERROR = "failed to simulate token payment",
+  SIGN_TRANSACTION_SUCCESS = "sign transaction: confirmed",
+  SIGN_TRANSACTION_FAIL = "sign transaction: rejected",
+  SIGN_TRANSACTION_MEMO_REQUIRED_FAIL = "sign transaction: memo required error",
+
+  // Asset Management Events
+  ADD_ASSET_SUCCESS = "manage asset: add asset",
+  ADD_TOKEN_SUCCESS = "manage asset: add token",
```

Comment:
nite: looks like this `manage asset: add toke` event is not being used on both mobile and extension - cc @danilosilvackl @aristidesstaffieri